core/src/main/scala/org/scalarelational/extra/PersistentProperties.scala

```diff
@@ -23,45 +25,45 @@ trait PersistentProperties extends Datastore {
 
   object persistence {
     def get(name: String): Option[String] = {
-      val query = select(persistentProperties.value) from persistentProperties where persistentProperties.key === name
+      val query = (select (persistentProperties.value)
+        from persistentProperties
+        where persistentProperties.key === name
+      )
+
       query.result.converted.headOption
     }
 
     def apply(name: String): String = {
-      get(name).getOrElse(throw new NullPointerException(s"Unable to find $name in persistent properties table."))
+      get(name).getOrElse(
+        throw new NullPointerException(s"Unable to find $name in persistent properties table."))
     }
 
-    def update(name: String, newValue: String) {
-      val m = merge(
+    def update(name: String, newValue: String): Int =
```

Comment:
Not relevant to this ticket, but this is going to have to change since not all databases support `Merge` apparently.